Source/Core/VideoCommon/PixelShaderManager.cpp

```diff
@@ -112,12 +88,20 @@ void PixelShaderManager::SetConstants()
 		dirty = true;
 		s_bViewPortChanged = false;
 	}
+
+	if (s_bEFBScaleChanged)
+	{
+		constants.efbscale[0] = 1.0f / float(Renderer::EFBToScaledXf(1));
+		constants.efbscale[1] = 1.0f / float(Renderer::EFBToScaledYf(1));
```

Comment:
Can this be moved into the SetChanged function? This Renderer call sounds like they don't need a delay (unlike xfmem). SetConstants is called quite often :/